src/js/components/SelectMultiple/__tests__/__snapshots__/SelectMultiple-test.tsx.snap

```diff
@@ -68,8 +68,8 @@ exports[`SelectMultiple children 1`] = `
   box-sizing: border-box;
   max-width: 100%;
   align-self: center;
-  margin-left: 12px;
-  margin-right: 12px;
+  margin-left: 16px;
```

Comment:
Changes only due to jsdom, does not appear in browser. Correct value is re-assigned below